### PR TITLE
docs: translate desafio01 to english

### PR DESCRIPTION
**Pull request by LauraBeatris**

### CODE REVIEW

**Comment by josepholiveira** [comment]
Hey,  @LauraBeatris.

First of all, thank you so much for your PR, i loved it 💜

But i think we can do some things to make it a little better, like creating the Notion page also in english, changing the screenshots to also be in english, and also adding a language selector in the main README of this repository.

If you agree to those changes, i would love to help in any way!

File: desafio-01/README.en.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  <a href="#memo-licence">Licence</a>
+</p>
+
+## :rocket: About the challenge
+
+In this challenge, you must plan your studies.
+
+For that, you must create a cronogram that will be adapted according to your time and rhythm.
+
+Define what'll be your study schedule, creating tasks that will be attached to a calendar, to help you keep focus and always aligned with your objectives!
```
Comment (josepholiveira) [inline desafio-01/README.en.md:35]: Change `to help you keep focus and always aligned with your objectives!` to `to help you study and always be aligned with your goals`

File: desafio-01/README.en.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+### Cronogram template
+
+To help you with that challenge, we created a template that you can duplicate through the Notion platform.
+Notion works either on mobile platforms or desktop platforms, and both can be downloaded in the [Oficial Page](https://www.notion.so/product).
```
Comment (josepholiveira) [inline desafio-01/README.en.md:40]: Change to `either on mobile platforms or desktop platforms`

File: desafio-01/README.en.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+📄 **[Link to the template](https://www.notion.so/Cronograma-de-estudos-e390bc8d2f5743668ec03348a3306070)**
+
+### Defining a weekly schedule
+
+<p align="center">
+  <img width="80%" src="./assets/cronograma-diario.png">
+</p>
+
+In this section you must inform a schedule that you will dedicate during your journey as a developer, after all, the studies routine will be the first step to you achieve your goals as quickly as possible!
```
Comment (josepholiveira) [inline desafio-01/README.en.md:52]: I think we could change this paragraph to this:

`
In this section you must inform your schedule, the time you will dedicate for your journey as a developer, after all, the studies routine will be your first step in order to achieve your goals as quickly as possible!
`

File: desafio-01/README.en.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+### Defining a daily schedule
+
+<p align="center">
+  <img width="80%" src="./assets/cronograma-semanal.png">
+</p>
+
+Here you must plan your daily tasks so you can be prepared to all the content that you'll consume during the week.
+
+You can follow the available template above or create your own study schedule, so you can always be organized and focused on your objectives.
```
Comment (josepholiveira) [inline desafio-01/README.en.md:62]: Change `objectives` to `goals`

File: desafio-01/README.en.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+
+We have already prepared everything that you can do in this first week of the bootcamp content, you just need to adapt it according to your routine.
+
+Tip: Add extra content like readings and podcasts, remember to have short breaks to rest to assimulate the material studied. Also add time to create content, such as posts on a blog or linkedin.
```
Comment (josepholiveira) [inline desafio-01/README.en.md:66]: Change `material` to `content`.

File: desafio-01/README.en.md (hunk 2 above)
Comment (LauraBeatris) [reply desafio-01/README.en.md:40]: But it's alright there, isn't it?

**Comment by josepholiveira** [comment]
Thats approved.

Thank you for your contribution, @LauraBeatris. You rock! 💜 🚀

**Pull request merged**